src/main/g8/build.sbt

```diff
@@ -69,4 +69,6 @@ libraryDependencies ++= Seq(
   "com.lightbend.akka" %% "akka-projection-testkit" % AkkaProjectionVersion % Test,
 )
 
-
+addCommandAlias("runNode1", "run local1.conf")
+addCommandAlias("runNode2", "run local2.conf")
+addCommandAlias("runNode3", "run local3.conf")
```

Comment:
I think we should call it `runNodeX` instead of `runLocalX`. 

The config is local, but we are starting a Node. Node is also the term we use for a Akka Cluster, well node.